Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Compilação e Executáveis\r\n*.exe\r\n*.out\r\n*.app\r\n*.o\r\n*.obj\r\n*.a\r\n*.lib\r\n*.so\r\n*.dll\r\n\r\n# Pastas de Build\r\n/cmake-build-debug/\r\n/cmake-build-release/\r\n/build/\r\n.cmake/\r\n\r\n# IDE - CLion / JetBrains\r\n.idea/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
--- a/.idea/.gitignore	(revision 5705011cba151ea92fc02d5517ed2ff4f94663d6)
+++ b/.idea/.gitignore	(date 1779239733336)
@@ -1,4 +1,5 @@
 # Compilação e Executáveis
+**/*.exe
 *.exe
 *.out
 *.app
